docs/ConfigureScalarDL.md

```diff
@@ -2,32 +2,39 @@
 
 This guide explains how to configure Scalar DL custom values in helm charts for Scalar DL deployment.
 
-## Create Kubernetes Secrets
+## Update Configuration Files
 
-Kubernetes Secret is an object that contains a small amount of sensitive data such as a password, a token, or a key. 
-This optional method is highly recommended for handling credentials in the production environment.
+### Kubernetes Secrets and Schema Loading reference
 
-You need to create a secret object with `db-username` and `db-password` to store the username and password of the underlying database.
+You can refer to the table below to get values to set create Kubernetes Secrets and update the [schema-loading-custom-values.yaml](../conf/schema-loading-custom-values.yaml) file properties file according to the database you chose for deployment.
```

Comment:
This sentence is not much clear `...update the schema-loading-custom-values.yaml file properties file according to the database you chose for deployment.`, please update it.